src/components/mortgage-investment-compare/input-card.js

```diff
@@ -301,7 +330,7 @@ const InputCard = props => {
             the US during the 1960s was roughly $20,000. The average home price
             today is roughly{' '}
             <a href='https://www.census.gov/construction/nrs/pdf/uspricemon.pdf'>
-              $370,000
+              $400,000
```

Comment:
lol what happened

Reply:
I just updated it with the most recent data. Average price of a house has increased since I first wrote this. We'll make it dynamic eventually.

Reply:
wow. yikes.